pkg/api/http/http_test.go

```diff
@@ -2499,6 +2499,7 @@ func TestV1Alpha1DistributedLock(t *testing.T) {
 
 		// assert
 		assert.Nil(t, resp.JSONBody)
+		assert.Contains(t, string(resp.RawBody), "ERR_MALFORMED_REQUEST")
```

Comment:
Add this line to ensure that I don't make any breaking changes to the error tag.